notifier.rst

```diff
@@ -87,6 +87,7 @@ Telnyx          ``symfony/telnyx-notifier``           ``telnyx://API_KEY@default
 TurboSms        ``symfony/turbo-sms-notifier``        ``turbosms://AUTH_TOKEN@default?from=FROM``
 Twilio          ``symfony/twilio-notifier``           ``twilio://SID:TOKEN@default?from=FROM``
 Yunpian         ``symfony/yunpian-notifier``          ``yunpian://APIKEY@default``
+KazInfoTeh      ``symfony/kazinfoteh-notifier``       ``kazinfoteh://username:password@default?sender=FROM``
```

Comment:
Please move above LightSms to keep alphanumeric order, thanks

Reply:
@fabpot I guess the correct package name would be:
```diff
- symfony/kazinfothe-notifier
+ symfony/kaz-info-the-notifier
```

Can you please confirm?

Reply:
The correct spelling would be `symfony/kaz-info-teh-notifier`. `teh` it's abbreviation from `technology` in Russian.

Reply:
Sorry ofc you are right I misspelled it 👍🏻